crates/holochain_conductor_api/src/app_interface.rs

```diff
@@ -76,6 +76,7 @@ pub enum AppResponse {
 
 /// The data provided over an app interface in order to make a zome call
 #[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "snake_case")]
```

Comment:
this wasn't really problematic anyways actually because it's a struct and the fields are in snake_case